cvs_gen.py

```diff
@@ -1,19 +1,24 @@
 import csv
-
 from database import view_data, get_all
+from helpers import number_format
+
+headers = view_data + ('2-th ph', '3-th ph',)
 
 path_output_file = "report.csv"
 
 with open(path_output_file, 'w') as csv_file:
-    writer = csv.DictWriter(csv_file, fieldnames=view_data,
-                            delimiter=';', lineterminator='\n')
+    writer = csv.DictWriter(
+        csv_file, fieldnames=headers, delimiter=',', lineterminator='\n')
     writer.writeheader()
     lines = get_all()
```

Comment:
You can drop this line and use for __line in get_all(), like in telegram reports